src/solvers/dgsem_tree/subcell_limiters_2d.jl

```diff
@@ -52,6 +56,172 @@ function (limiter::SubcellLimiterIDP)(u::AbstractArray{<:Any, 4}, semi, dg::DGSE
     return nothing
 end
 
+@inline function calc_bounds_2sided!(var_min, var_max, variable, u, t, semi)
+    mesh, equations, dg, cache = mesh_equations_solver_cache(semi)
+    # Calc bounds inside elements
+    @threaded for element in eachelement(dg, cache)
+        var_min[:, :, element] .= typemax(eltype(var_min))
+        var_max[:, :, element] .= typemin(eltype(var_max))
+        # Calculate bounds at Gauss-Lobatto nodes using u
+        for j in eachnode(dg), i in eachnode(dg)
+            var = u[variable, i, j, element]
+            var_min[i, j, element] = min(var_min[i, j, element], var)
+            var_max[i, j, element] = max(var_max[i, j, element], var)
+
+            if i > 1
+                var_min[i - 1, j, element] = min(var_min[i - 1, j, element], var)
+                var_max[i - 1, j, element] = max(var_max[i - 1, j, element], var)
+            end
+            if i < nnodes(dg)
+                var_min[i + 1, j, element] = min(var_min[i + 1, j, element], var)
+                var_max[i + 1, j, element] = max(var_max[i + 1, j, element], var)
+            end
+            if j > 1
+                var_min[i, j - 1, element] = min(var_min[i, j - 1, element], var)
+                var_max[i, j - 1, element] = max(var_max[i, j - 1, element], var)
+            end
+            if j < nnodes(dg)
+                var_min[i, j + 1, element] = min(var_min[i, j + 1, element], var)
+                var_max[i, j + 1, element] = max(var_max[i, j + 1, element], var)
+            end
+        end
+    end
+
+    # Values at element boundary
+    calc_bounds_2sided_interface!(var_min, var_max, variable, u, t, semi, mesh)
+end
+
+@inline function calc_bounds_2sided_interface!(var_min, var_max, variable, u, t, semi,
```

Comment:
```suggestion
@inline function calc_bounds_twosided_interface!(var_min, var_max, variable, u, t, semi,
```
As above

Reply:
Done in [5adf14c](https://github.com/trixi-framework/Trixi.jl/pull/1648/commits/5adf14c9d8bbb5cd043e36ad8067dfcd22880cb4)